### ISSUE TITLE
Hide a part of the option if the option width if greater than select width

### ISSUE THREAD

**Issue by C0ZEN** [silviomoreto/bootstrap-select#1186]
I didn't saw a thread about it but I've got an issue in small devices.
If my select have an option with a long text, the dropdown-menu's width is greater than the select button's width.
It's a problem with the small devices because the dropdown-menu's width is more important than the width of the page... The user need to scroll left-to-right to see the entire dropdown-menu.

I use the data-width at '100%' in a col-xs-12.
So, I want to know how can I hide a part of the right side of the option to reduce the width.
In other words, have my button's width equal to the dropdown-menu's width.

I've found something similar #1034 
I want to see the same result (with data-width="100%") but on for the entire dropdown-menu and not only for the button.

Thanks for the help.

**Comment by iyadthayyil** [comment]
fixed #1034
